tonic/src/codec/prost.rs

```diff
@@ -136,6 +136,37 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    async fn encode_too_big() {
+        let encoder = MockEncoder::default();
+
+        let msg = vec![0u8; u32::MAX as usize + 1];
```

Comment:
uh this seems like its failing on windows and I think that kinda makes sense when you try to allocate a vec of u32::MAX? Prob need to lower it for windows or something like that

Reply:
W/o that much data, we're not hitting the limit that triggers the bug. I can exclude the test for windows though.

Reply:
Yeah, lets see if that works, may just be some setting on windows that is making this fail.

Reply:
done